lnst/Recipes/ENRT/BaseTunnelRecipe.py

```diff
@@ -141,10 +143,11 @@ def get_packet_assert_config(self, ping_config):
         """
         raise NotImplementedError
 
-    def generate_perf_endpoints(self, config: TunnelConfiguration):
+    def generate_perf_endpoints(self, config: TunnelConfiguration) -> list[Collection[EndpointPair[IPEndpointLike]]]:
         """
         The perf endpoints for recipes derived from this class are usually
         the tunnel endpoints. The derived class can override the endpoints
         if needed.
         """
-        return [tuple(config.tunnel_devices)]
+        dev1, dev2 = config.tunnel_devices
+        return ip_endpoints(config, (dev1, dev2))
```

Comment:
this one is missing the top level list wrapper initializer - all of the other classes in this commit are changed to return:
```
[ip_endpoints(...)]
```
is this intentional here? The return type of the method is still the same.

Reply:
Definitely not, nice catch!